x/wasm/keeper/keeper.go

```diff
@@ -876,7 +876,7 @@ func (k Keeper) GetByteCode(ctx sdk.Context, codeID uint64) ([]byte, error) {
 	var codeInfo types.CodeInfo
 	codeInfoBz := store.Get(types.GetCodeKey(codeID))
 	if codeInfoBz == nil {
-		return nil, nil
+		return nil, types.ErrNotFound
```

Comment:
Please, add unittest in this line.

Reply:
I added the test.
Please could you check.